cylc/flow/cfgspec/globalcfg.py

```diff
@@ -166,7 +166,7 @@
 
         A workflow will stall if there are no tasks ready to run and no
         waiting external triggers, but the presence of incomplete
-        tasks or unsatisified prerequisites shows the workflow did not run to
+        tasks or unsatisfied prerequisites shows the workflow did not run to
```

Comment:
Damn it, that's probably me. When typing quickly, that combination of letters seems to come out of my fingers despite what my brain is saying.  :rofl:

Reply:
Suspect that we all do this sort of thing.